react/src/index.js

```diff
@@ -130,14 +139,20 @@ class App extends Component {
                 <Route exact path="/" component={Home} />
                 <Route path="/about" component={About} />
                 <Route path="/cart" component={Cart} />
-                <Route path="/checkout" component={Checkout} />
+                <Route path="/checkout">
+                  <Checkout backend={BACKEND_URL} history={history} />
+                </Route>
                 <Route path="/complete" component={Complete} />
                 <Route path="/error" component={CompleteError} />
                 <Route path="/cra" component={Cra} />
                 <SentryRoute path="/employee/:slug" component={Employee}></SentryRoute>
                 <SentryRoute path="/product/:id" component={Product}></SentryRoute>
-                <Route path="/products" component={Products} />
-                <Route path="/products-join" component={ProductsJoin} />
+                <Route path="/products">
```

Comment:
In general the proposed pattern here is, instead of calculating the backend URL in each of 3 individual files, [for example, here](https://github.com/sentry-demos/application-monitoring/blob/31ae8f6f7b946205e7d3dd88326a9fd799dd8b51/react/src/components/Products.js#L10-L16), we just calculate it once and pass it as `props` in our `Route` definitions.